tests/page/page-event-console.spec.ts

```diff
@@ -64,17 +64,21 @@ it('should work for different console API calls', async ({ page }) => {
     console.dir('calling console.dir');
     console.warn('calling console.warn');
     console.error('calling console.error');
+    console.info('calling console.info');
+    console.debug('calling console.debug');
```

Comment:
Actually, we can't use debug in WebKit - we use ut for bindings.

Reply:
Seems like we can? We emit custom information via console.debug but filter it out before we emit it to the client side so it seems fine?

https://github.com/microsoft/playwright/blob/f19864890fa6d4ecdadb2bba7def2f5bcde6bec5/packages/playwright-core/src/server/webkit/wkPage.ts#L495-L503

Reply:
Ah, you are right. Their inspector is busted, but we don't report our staff. Good to land then.

Reply:
Actually, let's add a binding to that test, otherwise, you land this test, I remove that branch from the code and test keeps passing.